var/spack/repos/builtin/packages/geant4/package.py

```diff
@@ -215,6 +215,7 @@ def std_when(values):
     # As released, 10.0.4 has inconsistently capitalised filenames
     # in the cmake files; this patch also enables cxxstd 14
     patch("geant4-10.0.4.patch", when="@10.0.4")
+    patch("columns.patch", when="@10.4:")
```

Comment:
Prefix with brief comment explaining what the patch does. Link to bugzilla report. Close version range assuming fixed in next release.

Reply:
```suggestion
    # Fix member field typo in g4tools wroot
    # See https://bugzilla-geant4.kek.jp/show_bug.cgi?id=2640
    patch("columns.patch", when="@10.4:11.2.2")
```